tests/test_ebs.py

```diff
@@ -432,6 +433,22 @@ def test_matched(self):
         assert perms == [{"UserId": "112233445566"}]
 
 
+class SnapshotVolumeFilter(BaseTest):
+
+    def test_ebs_volume_filter(self):
+        factory = self.replay_flight_data("test_ebs_volume_related_filter")
+        p = self.load_policy(yaml_load("""
+           name: ebs-snapshot-volume
+           resource: aws.ebs-snapshot
+           filters:
+             - type: volume
+               key: VolumeId
+               value: absent
+        """), session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
we generally always verify an attribute of the resource.

this use of absent on a related filter is probably  something that should get to the docs, else not sure its discoverable.

Reply:
agreed, can update the test for that